lease-kubernetes/src/main/scala/akka/coordination/lease/kubernetes/KubernetesLease.scala

```diff
@@ -87,21 +87,28 @@ class KubernetesLease private[akka] (system: ExtendedActorSystem, leaseTaken: At
 
   private val k8sSettings = KubernetesSettings(settings.leaseConfig, settings.timeoutSettings)
   private val leaseName = makeDNS1039Compatible(settings.leaseName, k8sSettings.allowLeaseNameTruncation)
+  private object Dispatchers {
+    implicit val blocking: ExecutionContext = system.dispatchers.lookup(DefaultBlockingDispatcherId)
+  }
+
+  private def k8sApiToken(): Future[String] = {
+    import Dispatchers.blocking
+    Future {
+      readConfigVarFromFilesystem(k8sSettings.apiTokenPath, "api-token").getOrElse("")
+    }
+  }
 
   private val k8sApi: Future[KubernetesApi] = {
-    implicit val blockingDispatcher: ExecutionContext = system.dispatchers.lookup(DefaultBlockingDispatcherId)
+    import Dispatchers.blocking
     for {
-      apiToken: String <- Future {
-        readConfigVarFromFilesystem(k8sSettings.apiTokenPath, "api-token").getOrElse("")
-      }
```

Comment:
Do we have the same problem in rolling-update-kubernetes and discovery-kubernetes-api?
Search for "api-token" to find those.

Reply:
Yes we do. There's a lot of repeat code here, should we factor it out into a common Kubernetes utility library?

Reply:
That was intentionally split up like that since the modules don't have dependencies to each other. Adding another module for this purpose might be worth it, or not.

Reply:
Done.